detectors/node/opentelemetry-resource-detector-aws/package.json

```
@@ -0,0 +1,68 @@
+{
+  "name": "@opentelemetry/resource-detector-aws",
+  "version": "0.23.0",
```

```suggestion
  "version": "0.24.0",
```
it will be 0.24.0 (or later) at the timer of merge.